eng/Baseline.Designer.props

```diff
@@ -324,7 +324,7 @@
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.JsonPatch' AND '$(TargetFramework)' == 'netstandard2.0' ">
     <BaselinePackageReference Include="Microsoft.CSharp" Version="[4.5.0, )" />
-    <BaselinePackageReference Include="Newtonsoft.Json" Version="[11.0.2, )" />
+    <BaselinePackageReference Include="Newtonsoft.Json" Version="[12.0.1, )" />
```

Comment:
This file is auto-generated, so this change will likely get reverted again. It's not really used in the master branch so doesn't matter much.